meetings/cpansec-minutes-2025-05-07.md

```diff
@@ -0,0 +1,250 @@
+toc: true
+meeting_time: 2025-05-07 16:00 UTC
+title: CPANSec bi-weekly minutes
+---
+
+## Agenda & Meeting Details 2025-05-07
+- 2025-05-07 16:00 UTC on #cpansec-discussion on Matrix
+
+
+## 15:40 UTC - Pre-meeting socializing
+-   Socializing & getting up to speed before the meeting starts properly
+-   Discuss organizing projects, swimlanes and issues (...)
+-   Check and resolve technical (A/V) issues before the meeting starts
+-   Come as you are!
+
+## 16:01 UTC - Meeting start
+
+## Welcome
+-   Meeting chair: @timlegge
+-   Meeting scribe: @sjn
+
+### Attendees, absents & regrets
+-   Attendees
+    - @timlegge, @sjn, @stigtsp, @toddr, @thibaultduponchelle, (@tux)
+-   Regrets 
+    -  @leont, @rrwo
+
+### Approve previous meeting minutes
+-   Previous meeting minutes was approved by @thibaultduponchelle and @stigtsp, and merged by @sjn
+
+## Agenda
+
+### Current matters
+
+#### CPAN Testers
+- [x] @tux - CPAN Testers work happening at PTS; We should contribute!
+  - e.g. ask if CPAN Testers can add results of Test::CVE
+
+#### XML::LibXML
+- [ ] @tux - XML::LibXML touches CVE issues in Alien
+  - @stigtsp - probably not a CVE for the CPAN module
+
+#### BSON::XS - vulnerable deps
+- [ ] @stigtsp - let's create a CVE
+
+
+### Operating changes
+- @sjn - Proposal: Add RSS feeds to the matrix channel? Suggestions for channel admins:
+   - Mastodon feed: https://fosstodon.org/@cpansec.rss
+   - News feed: https://security.metacpan.org/feed.xml
+   - CVE announcements feed: @stigtsp - can be done as part of an automated CVE announcement process
+   - CVE updates feed: Can hook right into the CVE.org database maybe
+- [ ] @stigtsp - good idea!
+
+
+### [Metadata & Software Bills of Materials](https://github.com/orgs/CPAN-Security/projects/1/views/2)
+
+#### CPAN::Meta::Spec, Requirements and PURLs
+- @sjn and @leont discussed at PTS, decided to
+  - [ ] @sjn - put together a detailed proposal
+
+#### CycloneDX 1.7 Sustainability fields
+- @sjn - Ongoing: project- & ecosystem-supplied status fields.
+  - @sjn - interesting discussions around exploring relationships between this spec and LF's security-insights.yml spec
+
+### [Compliance, Guidance & Standards](https://github.com/orgs/CPAN-Security/projects/9/views/2)
+
+#### CPAN Author's Security Policy Guidelines
+- [ ] @stigtsp - Minimal (TLDR) version of policy wanted
+- [ ] @tux - Team document template wanted; Help needed
+   - [ ] @timlegge has started on something; happy to discuss with @tux; PR in the works
+   - @robrwo - not so much a different kind of doc as possible different way to report vuln
+- [ ] @sjn - Alternative documents needed for non-Europeans (requested by jnap)
+   - @sjn - can help with a CRA perspective
+   - @sjn - ORC WG FAQ may help
+   - @robrwo - this may be misunderstanding of the purpose of the document
+   - @robrwo - the doc is best practice not specific to countries etc.
+- [ ] @robrwo mini-project to encourage the popular modules to add a security policy
```

Comment:
This was discussion from a previous meeting, so perhaps belongs in a section that identifies these as remaining issues?

Reply:
Yeah, this should have been removed; We didn't discuss it.

Reply:
Since I'm doing this; I've added my decision :sweat_smile: